service/rating.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'file_handler'
+
+# Description/Explanation of Rating module
+module Rating
+  def top_ids(n_times, orders, key_name)
+    orders.group_by { |object| object.instance_variable_get("@#{key_name}") }
```

Comment:
It's nice that you've investigated `instance_variable_get`, but why do you use it here? It's better to avoid using following constructions for now.

Reply:
Fixed. I've changed to object.send("#{key_name}"). Is it more acceptable?